server/data/vocabularies.json

```diff
@@ -0,0 +1,151 @@
+[
+    {
+        "_id": "subscriber_types",
+        "display_name": "Subscriber Types",
+        "type": "unmanageable",
+        "selection_type": "do not show",
+        "items": [
+            {
+                "is_active": true,
+                "name": "All",
+                "qcode": "all",
+                "formats": [
+                    {
+                        "name": "NINJS",
+                        "qcode": "ninjs"
+                    },
+                    {
+                        "name": "NITF",
+                        "qcode": "nitf"
+                    },
+                    {
+                        "name": "NewsML G2",
+                        "qcode": "newsmlg2"
+                    },
+                    {
+                        "name": "Email",
+                        "qcode": "email"
+                    },
+                    {
+                        "name": "Adobe InDesign (IDML)",
+                        "qcode": "idml"
+                    },
+                    {
+                        "name": "AAP NewsML 1.2",
```

Comment:
we should probably call it just `NewsML 1.2`

Reply:
@petrjasek ok